packages/terra-list/src/List.module.scss

```diff
@@ -135,4 +135,8 @@
   .item-end {
     flex: 0 0 auto;
   }
+
+  .list-fill {
+    width: 100%;
```

Comment:
why do we need this ...??

Reply:
list under `listsection` needed to fill its immediate parent https://github.com/cerner/terra-core/pull/3749/files#diff-322ed14e3702ee33c7a170797460a1d83e655b49641aea7ad3636db21f3d5dc4R79